mandatory/4-tax.js

```diff
@@ -4,9 +4,16 @@
   A business requires a program that calculates how much the price of a product is including sales tax
   Sales tax is 20% of the price of the product.
 */
-
-function calculateSalesTax() {}
-
+// price*product gives you the total price without tax.
+// 1 + salesTax gives you the price after tax.
+function calculateSalesTax(price,product) {
+  var salesTax = .20;
+  var totalPrice = (price * product) * (1+salesTax);
```

Comment:
I think all you need here is the "price" alone -- notice how in all your `console.log` tests you passed the price in as `1`, only varying the product. Then you can simply call this function as `calculateSalesTax(15)`.

What was supposed to be the purpose of `price` here? Were you thinking of quantity?